sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -43,171 +47,48 @@
-            }
-          ],
-          "content": "Welcome to your institution account!",
-          "refreshOnChange": false,
-          "customClass": "category-header-large primary-color",
-          "key": "html",
-          "type": "htmlelement",
-          "input": false,
-          "tableView": false,
-          "placeholder": "",
-          "prefix": "",
-          "suffix": "",
-          "multiple": false,
-          "defaultValue": null,
-          "protected": false,
-          "unique": false,
-          "persistent": false,
-          "hidden": false,
-          "clearOnHide": true,
-          "refreshOn": "",
-          "redrawOn": "",
-          "modalEdit": false,
-          "dataGridLabel": false,
-          "labelPosition": "top",
-          "description": "",
-          "errorLabel": "",
-          "tooltip": "",
-          "hideLabel": false,
-          "tabindex": "",
-          "disabled": false,
-          "autofocus": false,
-          "dbIndex": false,
-          "customDefaultValue": "",
-          "calculateValue": "",
-          "calculateServer": false,
-          "widget": null,
-          "attributes": {},
-          "validateOn": "change",
-          "validate": {
-            "required": false,
-            "custom": "",
-            "customPrivate": false,
-            "strictDateValidation": false,
-            "multiple": false,
-            "unique": false
-          },
-          "conditional": {
-            "show": null,
-            "when": null,
-            "eq": ""
-          },
-          "overlay": {
-            "style": "",
-            "left": "",
-            "top": "",
-            "width": "",
-            "height": ""
-          },
-          "allowCalculateOverride": false,
-          "encrypted": false,
-          "showCharCount": false,
-          "showWordCount": false,
-          "properties": {},
-          "allowMultipleMasks": false,
-          "addons": [],
-          "tag": "p",
-          "id": "ed5c94d"
-        },
         {
           "label": "Columns",
           "columns": [
             {
               "components": [
                 {
                   "label": "HTML",
+                  "labelWidth": "",
+                  "labelMargin": "",
+                  "tag": "div",
+                  "className": "",
                   "attrs": [
                     {
                       "attr": "",
                       "value": ""
                     }
                   ],
-                  "content": "<strong>New users</strong> can follow the steps listed below to get started.",
+                  "content": "<h1 class=\"category-header-large primary-color\">Welcome to your account!</h1>\n<p class=\"formio-text mt-4\">\n  <strong>New users</strong> can follow the steps listed below to get started.\n</p>\n<p class=\"formio-text\">\n  <strong>Returning users</strong> can use the left-hand navigation \nto do things like confirm enrolment or report a change.\n</p>",
```

Comment:
Why are we using H1 here ? category-header-large is setting the header font typography.

Reply:
![image](https://user-images.githubusercontent.com/77353155/186250174-749622a1-5209-4561-a01b-06a5f96ac0f5.png)
![image](https://user-images.githubusercontent.com/77353155/186250197-077ecd6d-03bb-4008-8339-796196143b0c.png)

Reply:
Yes of course styles will not change the html element tag. I was trying to understand if there was any specific advantage of using H1 here as html element's nature is going to be modified by the css class(H1 or div will produce same output). 
Representing a header content in H tag seems to valid purpose.